src/schema/v2/home/fetch.ts

```diff
@@ -12,18 +12,26 @@ import {
 import blacklist from "lib/artist_blacklist"
 import { LoadersWithAuthentication } from "lib/loaders/loaders_with_authentication"
 import { LoadersWithoutAuthentication } from "lib/loaders/loaders_without_authentication"
+import moment from "moment"
 
 export const featuredFair = (
   fairsLoader: LoadersWithoutAuthentication["fairsLoader"]
 ) => {
   return fairsLoader({
     size: 5,
     active: true,
+    sort: "-start_at",
     has_homepage_section: true,
   }).then(({ body: fairs }) => {
     if (fairs.length) {
+      const now = moment.utc()
+      const runningFairs = filter(fairs, (fair) => {
```

Comment:
**question:** Should this method return `undefined` if there are no running fairs (like it does on line 42)?

Reply:
That's a good point. I think what should still be returned in this case is `undefined`. For example, because of https://github.com/artsy/metaphysics/blob/1b0fe1d9f8554406e9139bfe04c1fd976077b10d/src/schema/v2/home/logged_out_modules.ts#L11 (if an empty array is returned, the check will pass)